pkg/plugin/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -58,7 +58,7 @@ const makefileTemplate = `
 # Image URL to use all building/pushing image targets
 IMG ?= {{ .Image }}
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
```

Comment:
Why we need to add `preserveUnknownFields=false` by default?

Reply:
#933. Also it is good practice in general, and can be removed if the user doesn't want webhooks.

Reply:
@estroz Is this really needed if controller-tools/controller-gen is defaulting to v1 CRDs now?
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#preserving-unknown-fields
```
For CustomResourceDefinitions created in apiextensions.k8s.io/v1, structural OpenAPI v3 validation schemas are required and pruning is enabled and cannot be disabled ...
```

Or is this mostly to be explicit in case the user sets `crd:crdVersions=v1beta1`?

Reply:
@hasbro17 yeah this is for `crd:crdVersions={v1beta1}`. Since `preserveUnknownFields=false` is a requirement for webhooks, setting this as the default ensures webhooks will work by default regardless of CRD version.